### ISSUE TITLE
continuation of run does not work with turbodonkey cart

### ISSUE THREAD

**Issue by koadman** [gjospin/PhyloSift#158]
when the sequences have been broken up into chunks for processing there is currently no means to restart a partially processed run. some means to determine what the last successful chunk of sequence to be processed would be required

**Comment by gjospin** [comment]
So here are my thoughts for this continuation.

We have to look at this on a chunk basis because we are carrying variables all the way from FastSearch to Summarize and restarting part way through wouldn't work.

Strategy 1:
launch_searches only when there is no sequence_taxa.$chunk.txt file with data in it.  If there is data in there, assume the chunk was completed.

Problem 1 : The chunk doesn't have any data to write and we would restart the chunk for nothing.
Problem 2 : If it fails during the writing of the sequence_taxa.$chunk.txt file, we would not restart the chunk even though it needs to be restarted.

Strategy2:
Maybe we can write to "run_info.txt" every time a chunk has completed and grep for "Chunk $chunk completed". If the result is empty, do that chunk.

Any thoughts ?

**Comment by koadman** [comment]
Yes, i think we need something like strategy 2.
I propose a pair of new Utilities functions:
something like `get_last_completed_chunk()` that returns a chunk ID
and something like `completed_chunk()` that stores the last completed chunk to a file, maybe ".last_chunk". Notice the leading . to make the file hidden to users.

**Comment by koadman** [comment]
storing which chunks were processed in the run_info.txt is a nice idea too. doing that right is a bit more involved though so we should make it a new issue for a future sprint.

**Comment by gjospin** [comment]
Here is what I did (committed and pushed). Lets see if it meets your standards/ideas 

I created a has_chuck_completed() function that looks up the chunk in the run_info.txt
I am appending the run_info so we keep track of what chunk finished as well as what command was ran on subsequent runs (it shows which DB and what command).  If it's not done right, I guess we can add that to next month's sprint to fix.
Closing this for now.
